server/tsuserver.py

```diff
@@ -679,7 +679,7 @@ def broadcast_need(self, client, msg):
         area_id = client.area.id
         self.send_all_cmd_pred('CT', '{}'.format(self.config['hostname']),
                                '=== Advert ===\r\n{} in {} [{}] needs {}\r\n==============='
-                               .format(char_name, area_name, area_id, msg), pred=lambda x: not x.muted_adverts)
+                                .format(char_name, area_name, area_id, msg), pred=lambda x: not x.muted_adverts)
```

Comment:
Remove extra space (Python PEP recommends multiline code is aligned with brackets)